packages/scipy/patches/make-int-return-values.patch

```diff
@@ -3,209 +3,2563 @@ is treated as returning void, when it really should return int (values are
+ extern void    super_stats (int, int *);
+diff --git a/scipy/sparse/linalg/dsolve/SuperLU/SRC/zgscon.c b/scipy/sparse/linalg/dsolve/SuperLU/SRC/zgscon.c
+index 8bb95aa..8b6651d 100644
+--- a/scipy/sparse/linalg/dsolve/SuperLU/SRC/zgscon.c
++++ b/scipy/sparse/linalg/dsolve/SuperLU/SRC/zgscon.c
+@@ -80,9 +80,6 @@ zgscon(char *norm, SuperMatrix *L, SuperMatrix *U,
+     int    kase, kase1, onenrm, i;
+     double ainvnm;
+     doublecomplex *work;
+-    extern int zrscl_(int *, doublecomplex *, doublecomplex *, int *);
+-
+-    extern int zlacon_(int *, doublecomplex *, doublecomplex *, double *, int *);
+ 
+     
+     /* Test the input parameters. */
+diff --git a/scipy/sparse/linalg/dsolve/SuperLU/SRC/zgsequ.c b/scipy/sparse/linalg/dsolve/SuperLU/SRC/zgsequ.c
+index 657637d..b04e46f 100644
+--- a/scipy/sparse/linalg/dsolve/SuperLU/SRC/zgsequ.c
++++ b/scipy/sparse/linalg/dsolve/SuperLU/SRC/zgsequ.c
+@@ -92,7 +92,6 @@ zgsequ(SuperMatrix *A, double *r, double *c, double *rowcnd,
+     int i, j, irow;
+     double rcmin, rcmax;
+     double bignum, smlnum;
+-    extern double dlamch_(char *);
+     
+     /* Test the input parameters. */
+     *info = 0;
+diff --git a/scipy/sparse/linalg/dsolve/SuperLU/SRC/zgsrfs.c b/scipy/sparse/linalg/dsolve/SuperLU/SRC/zgsrfs.c
+index 2ed0481..110bfe1 100644
+--- a/scipy/sparse/linalg/dsolve/SuperLU/SRC/zgsrfs.c
++++ b/scipy/sparse/linalg/dsolve/SuperLU/SRC/zgsrfs.c
+@@ -158,14 +158,6 @@ zgsrfs(trans_t trans, SuperMatrix *A, SuperMatrix *L, SuperMatrix *U,
+     double   *rwork;
+     int      *iwork;
+ 
+-    extern int zlacon_(int *, doublecomplex *, doublecomplex *, double *, int *);
+-#ifdef _CRAY
+-    extern int CCOPY(int *, doublecomplex *, int *, doublecomplex *, int *);
+-    extern int CSAXPY(int *, doublecomplex *, doublecomplex *, int *, doublecomplex *, int *);
+-#else
+-    extern int zcopy_(int *, doublecomplex *, int *, doublecomplex *, int *);
+-    extern int zaxpy_(int *, doublecomplex *, doublecomplex *, int *, doublecomplex *, int *);
+-#endif
+ 
+     Astore = A->Store;
+     Aval   = Astore->nzval;
+diff --git a/scipy/sparse/linalg/dsolve/SuperLU/SRC/zlacon.c b/scipy/sparse/linalg/dsolve/SuperLU/SRC/zlacon.c
+index b2cd1ed..6934f1e 100644
+--- a/scipy/sparse/linalg/dsolve/SuperLU/SRC/zlacon.c
++++ b/scipy/sparse/linalg/dsolve/SuperLU/SRC/zlacon.c
+@@ -85,9 +85,6 @@ zlacon_(int *n, doublecomplex *v, doublecomplex *x, double *est, int *kase)
+     static int i, j;
+     double temp;
+     double safmin;
+-    extern double dlamch_(char *);
+-    extern int izmax1_(int *, doublecomplex *, int *);
+-    extern double dzsum1_(int *, doublecomplex *, int *);
+ 
+     safmin = dlamch_("Safe minimum");
+     if ( *kase == 0 ) {
+diff --git a/scipy/sparse/linalg/dsolve/SuperLU/scipychanges.patch b/before/scipy/sparse/linalg/dsolve/SuperLU/scipychanges.patch
+deleted file mode 100644
+index 7a0a3ba..0000000
+--- a/scipy/sparse/linalg/dsolve/SuperLU/scipychanges.patch
++++ /dev/null
```

Comment:
@joemarshall why is this patch removed?

Reply:
I reverted the removal for now. Let's see if it will an issue in https://github.com/iodide-project/pyodide/pull/1102 Thanks for the feedback.

Reply:
Weird, I can't for the life of me remember why I killed that. I think it was possibly accidental when I was going through superlu and fixing the missing externs. I think that patch may not be ever called, I think it just is there to show the diff between scipy and base superlu...